app/src/main/cpp/src/3rd_party/android-helpers/openssl/build_openssl.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash \
+
+ANDROID_NDK=$1
+API_LEVEL=23
+BUILD_DIR=$2
+OUT_DIR=$3
+
+##### Prepare Files #####
+sed -i 's/.*-mandroid.*//' Configurations/15-android.conf
+patch -p1 -N <<EOP
```

Comment:
@VadymLuchko suggest to move out this patch into a separate file in the same folder

Reply:
done in 048ae59c19ca4e61f504361cf27bd9190b2e5830